src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -100,6 +127,24 @@ def dcos_browse(name, resource_group_name, disable_browser=False):
 
     return
 
+def acs_install_cli(resource_group, name, install_location=None, client_version=None):
+    acs_info = _get_acs_info(name, resource_group)
+    orchestrator_type = acs_info.orchestrator_profile.orchestrator_type  # pylint: disable=no-member
+    if  orchestrator_type == 'kubernetes':
+        # TODO: is there a more python idiomatic way to do this?
+        if client_version:
+            return k8s_install_cli(client_version=client_version, install_location=install_location)
+        else:
+            return k8s_install_cli(install_location=install_location)
+    elif orchestrator_type == 'dcos':
+        # TODO: is there a more python idiomatic way to do this?
+        if client_version:
+            return dcos_install_cli(client_version=client_version, install_location=install_location)
+        else:
+            return dcos_install_cli(install_location=install_location)
```

Comment:
You could say:
```Python
return dcos_install_cli(client_version=client_version, install_location=install_location) if client_version else dcos_install_cli(install_location=install_location)
```

Also, I think if you created a kwargs dictionary, you could just say:

```Python
kwargs = {'install_location': install_location}
if client_version:
  kwargs['client_version'] = client_version
return dcos_install_cli(**kwargs)
```